### ISSUE TITLE
How to set that backups are never deleted?

### ISSUE THREAD

**Issue by coezbek** [marvinweber/KPSimpleBackup#38]
I would have assumed that putting 0 into the number of backups to keep does this, but it isn't mentioned in the docs.

Can you confirm?

**Comment by marvinweber** [comment]
No, in the current implementation setting this to `0` will delete all backups.  
This obviously does not make sense, it should be changed to either not delete any backups when setting to `0` or providing a checkbox and not allowing a setting of `0`.

In the meantime, as a workaround, you can set the limit to a very high number you'd probably never reach...
